### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Added Login and SignUp screens with routes

### CODE REVIEW

File: src/Page/LoginPage/Login.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    display: flex;
+    justify-content: flex-start;
+    margin: 0.7rem;
+}
+
+.remember-me{
+    margin-left: -4rem;
+}
+.forgotPassword{
+    color: #288cfb;
```
Comment (atharvabhanage02) [inline src/Page/LoginPage/Login.css]: use color variables here

File: src/Page/LoginPage/Login.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import "./Login.css"
+const Login = () => {
+  return (
+    // <div className="outer-product-side-bar">
```
Comment (atharvabhanage02) [inline src/Page/LoginPage/Login.js]: remove commented code before you merge

**Review by atharvabhanage02 — APPROVED** [review]
Just few suggestions , else 
- UI looks clean 
- Code is well structured

File: src/Page/LoginPage/Login.js (hunk 2 above)
Comment (neetukumari4858) [reply src/Page/LoginPage/Login.js]: okey I will do

**Pull request merged**